doc/examples/visualization.ipynb

```diff
@@ -0,0 +1,326 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "petab_select.plot.scatter_criterion_vs_n_estimated(\n",
```

Comment:
In the current example, the points overlap quite a bit. I guess this will happen rather frequently. Some [jitter](https://seaborn.pydata.org/generated/seaborn.objects.Jitter.html#seaborn.objects.Jitter) could be helpful here.